x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/AuthorizationService.java

```diff
@@ -320,7 +320,7 @@ public void authorize(Authentication authentication, String action, TransportReq
             // only the XPackUser is allowed to work with this index, but we should allow indices monitoring actions through for debugging
             // purposes. These monitor requests also sometimes resolve indices concretely and then requests them
             logger.debug("user [{}] attempted to directly perform [{}] against the security index [{}]",
-                authentication.getUser().principal(), action, SecurityIndexManager.SECURITY_INDEX_NAME);
+                authentication.getUser().principal(), action, SystemIndicesNames.SECURITY_INDEX_NAME);
             throw denial(auditId, authentication, action, request, permission.names());
```

Comment:
@jaymode should we keep this behavior of granting monitoring . A test is failing:`AuthorizationServiceTests.testGrantedNonXPackUserCanExecuteMonitoringOperationsAgainstSecurityIndex` and I am not sure how to fix this.

Reply:
If we make this 7.0 only, then I think we can live with not granting monitoring implicitly but requiring it to be listed explicitly